### PR TITLE
Update the `standard library` term

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Update the `standard library` term.

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: docs/adding-a-new-ballerina-module.md [hunk 1]
```diff
@@ -44,15 +44,15 @@ This is a step-by-step guide on creating a new Ballerina module. It will guide y
         * 6.11.8 [The `settings.gradle` file](#the-settingsgradle-file-required)
         * 6.11.9 [The `spotbugs-exclude.xml` file](#the-spotbugs-excludexml-file-optional)
 7. [Add the new module](#add-the-new-module)
-    * 7.1 [Add the module to the Ballerina Standard Library](#add-the-module-to-the-ballerina-standard-library-optional)
+    * 7.1 [Add the module to the Ballerina library](#add-the-module-to-the-ballerina-standard-library-optional)
```
Comment (ThisaruGuruge) [inline docs/adding-a-new-ballerina-module.md:47]: Anchor will break here. Have to update that as well.

File: docs/adding-a-new-ballerina-module.md [hunk 2]
```diff
@@ -44,15 +44,15 @@ This is a step-by-step guide on creating a new Ballerina module. It will guide y
...
     * 7.2 [Add the module to Ballerina daily full build pipeline](#add-the-module-to-ballerina-daily-full-build-pipeline-required)
     * 7.3 [Add the module to Ballerina distribution](#add-the-module-to-ballerina-distribution-optional)
 
 ## Introduction
 
 This guide helps you understand the file structure, build scripts, and workflow scripts of a Ballerina module. It also explains how to add a new module to the Ballerina daily build and release pipelines.
 
-Examples of the directory structure and common files can be found in existing Ballerina Standard Library module repositories. The links to these repositories are available on the [Ballerina Standard Library Dashboard](https://github.com/ballerina-platform/ballerina-standard-library#status-dashboard).
+Examples of the directory structure and common files can be found in the existing Ballerina library module repositories. The links to these repositories are available on the [Ballerina Library Dashboard](https://github.com/ballerina-platform/ballerina-standard-library#status-dashboard).
```
Comment (ThisaruGuruge) [inline docs/adding-a-new-ballerina-module.md:55]: We can update the link here.
```suggestion
Examples of the directory structure and common files can be found in the existing Ballerina library module repositories. The links to these repositories are available on the [Ballerina Library Dashboard](https://github.com/ballerina-platform/ballerina-library#status-dashboard).
```

File: docs/adding-a-new-ballerina-module.md [hunk 3]
```diff
@@ -428,7 +428,7 @@ This field defines the name of the module. This should be the name of the reposi
 
 The version key of the module. This is related to the version prefix mentioned in the [`gradle.properties`](#the-gradleproperties-file-required) file. (e.g., `ballerinaStdlibIo`). It is used to add the module as a dependency to another repository including the [`ballerina-distribution`](https://github.com/ballerina-platform/ballerina-distribution) repository.
 
-Refer the [`version_key`](#the-versionkey-field) section under [Add the module to the Ballerina Standard Library](#add-the-module-to-the-ballerina-standard-library-optional). 
+Refer the [`version_key`](#the-versionkey-field) section under [Add the module to the Ballerina library](#add-the-module-to-the-ballerina-standard-library-optional).
```
Comment (ThisaruGuruge) [inline docs/adding-a-new-ballerina-module.md:431]: This anchor too will be broken.

File: docs/build-and-release-guide.md [hunk 4]
```diff
@@ -130,15 +130,15 @@ Checklist for the release,
...
 
 ### Multiple Module Release
 
-The Standard Library Release Manager will use [Stdlib Release Workflow](https://github.com/ballerina-platform/ballerina-standard-library/actions/workflows/release_pipeline.yml) to release multiple modules(Usually during Ballerina Update releases). It will publish all modules in the [list](https://github.com/ballerina-platform/ballerina-standard-library/blob/main/release/resources/stdlib_modules.json). The Release Manager can override it by using the `release` property.
+The Ballerina Library Release Manager will use [Stdlib Release Workflow](https://github.com/ballerina-platform/ballerina-standard-library/actions/workflows/release_pipeline.yml) to release multiple modules(Usually during Ballerina Update releases). It will publish all modules in the [list](https://github.com/ballerina-platform/ballerina-standard-library/blob/main/release/resources/stdlib_modules.json). The Release Manager can override it by using the `release` property.
```
Comment (ThisaruGuruge) [inline docs/build-and-release-guide.md:141]: Links can be updated here.

File: docs/adding-a-new-ballerina-module.md [hunk 5]
```diff
@@ -381,11 +381,11 @@ This file is used to skip specific spotbugs warnings/errors. This is required on
...
 
-### Add the module to the Ballerina Standard Library [Optional]
+### Add the module to the Ballerina library [Optional]
 
-This step is required only if the module is a part of the Ballerina Standard Library and if it should be added to the Ballerina Standard Library Dashboard and release process.
+This step is required only if the module is a part of the Ballerina library and if it should be added to the Ballerina library dashboard and release process.
 
-To add a module to the Ballerina Standard Library, add an entry in the [`module_list.json`](https://github.com/ballerina-platform/ballerina-standard-library/blob/main/release/resources/module_list.json) file in the [`ballerina-standard-library`](https://github.com/ballerina-platform/ballerina-standard-library) repository.
+To add a module to the Ballerina library, add an entry in the [`module_list.json`](https://github.com/ballerina-platform/ballerina-standard-library/blob/main/release/resources/module_list.json) file in the [`ballerina-standard-library`](https://github.com/ballerina-platform/ballerina-standard-library) repository.
```
Comment (NipunaRanasinghe) [inline docs/adding-a-new-ballerina-module.md:388]: Need to update the links in here as well
```suggestion
To add a module to the Ballerina library, add an entry in the [`module_list.json`](https://github.com/ballerina-platform/ballerina-library/blob/main/release/resources/module_list.json) file in the [`ballerina-library`](https://github.com/ballerina-platform/ballerina-library) repository.
```

File: docs/adding-a-new-ballerina-module.md [hunk 6]
```diff
@@ -405,9 +405,9 @@ This field is optional. It is only required if the version key cannot be inferre
 
 >**Note:** The version key is case-sensitive and should be in the camel case format.
 
-Once the `module_list.json` file is updated, the [`Update Stdlib Dependency Graph`](https://github.com/ballerina-platform/ballerina-standard-library/actions/workflows/update_dependencies.yml) workflow will run automatically to update the Standard Library Dashboard.
+Once the `module_list.json` file is updated, the [`Update Stdlib Dependency Graph`](https://github.com/ballerina-platform/ballerina-standard-library/actions/workflows/update_dependencies.yml) workflow will run automatically to update the Ballerina library Dashboard.
```
Comment (NipunaRanasinghe) [inline docs/adding-a-new-ballerina-module.md:408]:
```suggestion
Once the `module_list.json` file is updated, the [`Update Library Dependency Graph`](https://github.com/ballerina-platform/ballerina-library/actions/workflows/update_dependencies.yml) workflow will run automatically to update the Ballerina library Dashboard.
```

File: docs/build-and-release-guide.md [hunk 7]
```diff
@@ -162,21 +162,21 @@ The Standard Library Release Manager will use [Stdlib Release Workflow](https://
...
 4. Ensure Ballerina Release Manager has released all modules to the Central Staging after each RC vote. The release Manager of the Ballerina Release will publish this. Module owners are responsible for the ballerinax components.
-5. Standard Library release manager runs the [stdlib test workflow](https://github.com/ballerina-platform/ballerina-standard-library/actions/workflows/test_stdlib_releases_with_staging.yml) to verify newly published modules are working in an integration scenario. (This is to ensure no standard library release will break the existing users' build)
+5. The Ballerina library release manager runs the [stdlib test workflow](https://github.com/ballerina-platform/ballerina-standard-library/actions/workflows/test_stdlib_releases_with_staging.yml) to verify newly published modules are working in an integration scenario. (This is to ensure no Ballerina library release will break the existing users' build)
```
Comment (NipunaRanasinghe) [inline docs/build-and-release-guide.md:169]:
```suggestion
5. The Ballerina library release manager runs the [stdlib test workflow](https://github.com/ballerina-platform/ballerina-library/actions/workflows/test_stdlib_releases_with_staging.yml) to verify newly published modules are working in an integration scenario. (This is to ensure no Ballerina library release will break the existing users' build)
```

File: docs/build-and-release-guide.md [hunk 8]
```diff
@@ -162,21 +162,21 @@ The Standard Library Release Manager will use [Stdlib Release Workflow](https://
...
 6. Once the final RC pack is passed the voting phase, update the [module list](https://github.com/ballerina-platform/ballerina-standard-library/blob/main/release/resources/stdlib_modules.json) to reflect the modules to be released.
    >**Note:** If a module is not releasing with the given distribution release, change the `release` property to `false` in the module list. (This can be done on a separate branch and that branch can be used to trigger the release workflow.)
 7. Once the module list is updated, notify the team to refrain from merging any PRs to the default branches, including the post-release PRs.
 8. Then the release manager can trigger the [Stdlib Release Workflow](https://github.com/ballerina-platform/ballerina-standard-library/actions/workflows/release_pipeline.yml) to trigger the release.
```
Comment (NipunaRanasinghe) [inline docs/build-and-release-guide.md:173]:
```suggestion
8. Then the release manager can trigger the [Stdlib Release Workflow](https://github.com/ballerina-platform/ballerina-library/actions/workflows/release_pipeline.yml) to trigger the release.
```

File: docs/adding-a-new-ballerina-module.md (hunk 1 above)
Comment (praneesha) [reply docs/adding-a-new-ballerina-module.md:47]: Updated.

File: docs/adding-a-new-ballerina-module.md (hunk 2 above)
Comment (praneesha) [reply docs/adding-a-new-ballerina-module.md:55]: Fixed.

File: docs/adding-a-new-ballerina-module.md (hunk 3 above)
Comment (praneesha) [reply docs/adding-a-new-ballerina-module.md:431]: Updated.

File: docs/build-and-release-guide.md (hunk 4 above)
Comment (praneesha) [reply docs/build-and-release-guide.md:141]: Updated.

**Review by ThisaruGuruge — APPROVED** [review]
LGTM

**Pull request merged**